src/SonarQube.Client.RequestGenerator/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading;
+using SonarQube.Client.Api.Requests;
+using SonarQube.Client.Services;
+
+namespace SonarQube.Client.RequestGenerator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var requestFactory = new RequestFactory();
+            DefaultConfiguration.Configure(requestFactory);
+
+            var cts = new CancellationTokenSource();
+
+            var service = new SonarQubeService2(new HttpClientHandler(), requestFactory);
+            //var service = new SonarQubeService(new SonarQubeClientFactory());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS125)